teacher.rb

```diff
@@ -0,0 +1,12 @@
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(age, _parent_permission, specialization, name = 'Unkown')
```

Comment:
- [ ] As requested by the previous reviewer,  the parameter 'parent_permission' should be optional and have a default value of 'true'.